students_proj/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,5 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.5
+org.eclipse.jdt.core.compiler.compliance=1.5
+org.eclipse.jdt.core.compiler.problem.forbiddenReference=warning
+org.eclipse.jdt.core.compiler.source=1.5
```

Comment:
The project, as well as the simple smarthome project, have to be Java 1.8 compliant. Please use Eclipse IDE settings to change this for both projects, thus updating the jdr.core.prefs files indirectly.